### PR TITLE
add notes about contributing

### PR DESCRIPTION
**Pull request by glennj**
To reduce the confusion as seen in, for example, #111

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]
Excellent idea

File: README.md [hunk 1]
```diff
@@ -2,9 +2,18 @@
...
+Exercism exercises in [8th](https://8th-dev.com/manual.html).
 
-## Testing
+## CONTRIBUTING
+
+### Caution: CI will fail for PRs coming from forked repos
+
+This track [uses its own test runner for CI](https://github.com/exercism/8th/blob/1176bfbd9457f51556c77b8cc26408d718d1b4d6/.github/workflows/test.yml#L18).
+This requires a secret token to grant access.
+I'm not sure how this mechanism works, but it means that, for PRs originating from _a fork of this repo_, access cannot be granted and CI will fail: CI will only pass for PRs merging an exercism/8th branch.
```
Comment (ErikSchierboom) [inline README.md:13]: Maybe remove the I'm not sure part?

Comment (glennj) [reply README.md:13]: Is the rest of it ("it means that ...") more or less correct?

Comment (ErikSchierboom) [reply README.md:13]: It is. Forks don't have access to secrets for security reasons, hence the workflow thinking there is no token provided, as that is using a secret.

**Pull request merged**